src/server/lib/__tests__/middleware/csrf.test.ts

```diff
@@ -11,16 +11,18 @@ const testToken =
 // moved mocks before importing csrf module
 // to avoid "ReferenceError: Cannot access 'digestSpy' before initialization"
 const digestSpy = jest.fn();
-const randomBytesSpy = jest.fn();
+const randomBytesSpy = jest
+	.fn()
+	.mockReturnValue(Buffer.from(testToken, 'base64url'));
+
+// Mock crypto module
 jest.mock('crypto', () => ({
 	createHmac: () => ({
 		update: () => ({
 			digest: digestSpy.mockReturnValue('hash'),
 		}),
 	}),
-	randomBytes: randomBytesSpy.mockReturnValue(
-		Buffer.from(testToken, 'base64url'),
-	),
```

Comment:
This was changed as we were getting an

`ReferenceError: Cannot access 'randomBytesSpy' before initialization`

error.